### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/de/app.json [hunk 1]
```diff
@@ -314,17 +300,63 @@
...
+            },
+            "learnMore": "Weitere Details zu den Zahlungsmethoden findest du hier",
+            "title": "Zahlungsmethoden"
+          },
+          "supportedCurrencies": "Unterstützt alle gängigen Fiat-Währungen: USD, EUR, CHF und mehr."
+        },
+        "pocket": {
+          "fee": "Gebühr: {{fee}}%",
+          "learnMore": "Erfahre mehr über Pocket",
+          "supportedBankTransfers": "Unterstützt SEPA-Sofortüberweisungen in EUR und Sepa-Überweisungen in CHF"
```
Comment (thisconnect) [inline frontends/web/src/locales/de/app.json:338]: @benma this should change with https://github.com/digitalbitbox/bitbox-wallet-app/pull/1994

Comment (benma) [reply frontends/web/src/locales/de/app.json:338]: updated

Comment (thisconnect) [reply frontends/web/src/locales/de/app.json:338]: thank you

Comment (thisconnect) [reply frontends/web/src/locales/de/app.json:338]: @benma I don't see them

File: frontends/web/src/locales/de/app.json [hunk 2]
```diff
@@ -314,17 +300,61 @@
...
+              "cards": "Amex, Mastercard, Visa und Maestro"
+            },
+            "learnMore": "Weitere Details zu den Zahlungsmethoden findest du hier",
+            "title": "Zahlungsmethoden"
+          },
+          "supportedCurrencies": "Unterstützt alle gängigen Fiat-Währungen: USD, EUR, CHF und mehr."
+        },
+        "pocket": {
+          "learnMore": "Erfahre mehr über Pocket"
+        },
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/de/app.json:357]: ping!

**Pull request merged**